app/src/main/java/org/dosomething/letsdothis/ui/adapters/CampaignAdapter.java

```diff
@@ -110,8 +114,11 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, final int pos
             CampaignViewHolder campaignViewHolder = (CampaignViewHolder) holder;
             campaignViewHolder.title.setText(campaign.title);
             campaignViewHolder.callToAction.setText(campaign.callToAction);
-            Picasso.with(campaignViewHolder.imageView.getContext()).load(campaign.imagePath)
-                    .into(campaignViewHolder.imageView);
+            if(!TextUtils.isEmpty(campaign.imagePath))
+            {
+                Picasso.with(campaignViewHolder.imageView.getContext()).load(campaign.imagePath)
+                       .into(campaignViewHolder.imageView);
```

Comment:
do a resize for all Picasso requests because otherwise the images are too big and load time are long and cache get full quicker...